src/guard.rs

```diff
@@ -100,7 +100,7 @@ pub fn Any<F: Guard + 'static>(guard: F) -> AnyGuard {
     AnyGuard(vec![Box::new(guard)])
 }
 
-/// Matches if any of supplied guards matche.
+/// Matches if any of supplied guards match.
```

Comment:
I think it's clearer:

```suggestion
/// Matches any of supplied guards.
```